var/spack/repos/builtin/packages/fdb/package.py

```diff
@@ -42,6 +42,7 @@ class Fdb(CMakePackage):
 
     depends_on("eckit@1.16:")
     depends_on("eckit+admin", when="+tools")
+    depends_on("eckit@1.24.4:", when="@5.11.22:")
```

Comment:
```suggestion
    depends_on("eckit@1.16:")
    depends_on("eckit@1.24.4:", when="@5.11.22:")
    depends_on("eckit+admin", when="+tools")
```

I'd swap the last two lines here so that logically the variant dependency is last and not breaking up the two version constrained dependencies.

Reply:
makes perfect sense, thanks!